mockwebserver/src/main/kotlin/okhttp3/mockwebserver/MockWebServer.kt

```diff
@@ -1026,6 +1026,13 @@ class MockWebServer : ExternalResource(), Closeable {
         readBody = true
       }
 
+      peek.informationalResponses.forEach {
```

Comment:
FWIW, I dislike `forEach` as an alternative to `for(...)`. It’s needlessly complex to step-debug!